packages/react-native-web/src/exports/Modal/ModalPortal.js

```diff
@@ -16,7 +16,7 @@ export type ModalPortalProps = {|
   children: any
 |};
 
-function ModalPortal(props: ModalPortalProps) {
+function ModalPortal(props: ModalPortalProps): any | null {
```

Comment:
`any | null` is a pointless type

Reply:
Need to look at this one more, this may need some react-dom types imported as createPortal is currently any

Reply:
Imported ReactDOM types